### PR TITLE
Localize multiple choice answers, bookmarked songs, recently added

### LINKED ISSUE #1195: Use hangul song names as multiple choice options when using Korean locale

### PR DESCRIPTION
**Pull request by taahamahdi**
Closes #1195.

### CODE REVIEW

File: src/helpers/game_utils.ts [hunk 1]
```diff
@@ -240,18 +245,18 @@ export async function getMultipleChoiceOptions(
         guessMode === GuessModeType.SONG_NAME ||
         guessMode === GuessModeType.BOTH
     ) {
+        const songName = useHangul ? "hangul_song_name" : "clean_song_name";
         easyNames = (
             await dbContext
                 .kmq("available_songs")
-                .select("clean_song_name")
-                .groupByRaw("UPPER(clean_song_name)")
+                .select(songName)
```
Comment (Brainicism) [inline src/helpers/game_utils.ts:252]: Do all songs have a hangul_song_name?

**Review by Brainicism — CHANGES_REQUESTED** [review]

File: src/helpers/game_utils.ts (hunk 1 above)
Comment (taahamahdi) [reply src/helpers/game_utils.ts:252]: Many songs don't have a hangul_song_name

Comment (Brainicism) [reply src/helpers/game_utils.ts:252]: Won't this end up with a lot of empty strings in multiple choice + korean locale

Comment (taahamahdi) [reply src/helpers/game_utils.ts:252]: I can add a filter for checking they're non-empty. Then, they should be populated with easy choices below

**Review by Brainicism — DISMISSED** [review]

**Review by Brainicism — APPROVED** [review]

**Pull request merged**